src/handlers/motions/motionVoteSubmitted/index.ts

```diff
@@ -0,0 +1,40 @@
+import { ContractEvent } from '~types';
+import { getVotingClient, verbose } from '~utils';
+import {
+  getMotionDatabaseId,
+  getMotionFromDB,
+  updateMotionInDB,
+} from '../helpers';
+import { getUpdatedVoterRecord } from './helpers';
+
+export default async (event: ContractEvent): Promise<void> => {
+  const {
+    contractAddress: colonyAddress,
+    args: { motionId, voter },
+  } = event;
+
+  const votingClient = await getVotingClient(colonyAddress);
+  const { chainId } = await votingClient.provider.getNetwork();
+  const motionDatabaseId = getMotionDatabaseId(
+    chainId,
+    votingClient.address,
+    motionId,
+  );
+  const votedMotion = await getMotionFromDB(colonyAddress, motionDatabaseId);
+
+  if (votedMotion) {
```

Comment:
Do we want to log when that's not found?

Reply:
@jakubcolony It gets logged inside `getMotionFromDB`